apps/daimo-mobile/src/view/screen/send/SendTransferScreen.tsx

```diff
@@ -255,9 +256,19 @@ function RecipientDisplay({
   const { originalMatch } = recipient;
   const showOrig = originalMatch != null && originalMatch !== disp;
 
+  const nav = useNav();
+  const goToAccount = useCallback(() => {
+    nav.navigate("HomeTab", {
```

Comment:
This transition from send tab -> home tab feels a bit weird in the video, thoughts on duping the screen in send tab? cc @michelle-chiu as well

Reply:
video is edited to fit in the GH 10mb limit

Reply:
i see what you mean though--you're right, i think we can keep the nav smoother that way

Reply:
done

Reply:
@dcposch @nalinbhardwaj - How would we feel about removing the second screen entirely? 

Other than that, looking great so far! Thanks for taking care of this so quickly.

Reply:
what do you mean by the second screen?

Reply:
Also, instead of making "View on Block Explorer" the blue link style, let's keep it consistent with other parts of the app by making it a secondary button style (seen here): 

<img width="699" alt="Screenshot 2023-12-10 at 7 59 02 PM" src="https://github.com/daimo-eth/daimo/assets/5884103/6a54e185-1d10-4667-95fa-fd3b2bb45042">

Figma link: https://www.figma.com/file/wBBNeEzlsObaE8A8i9iVf7/Mobile-App?type=design&node-id=2403%3A14028&mode=design&t=TtmgbNcOvSyhvvXU-1

Reply:
Sounds good

<img width="250" alt="image" src="https://github.com/daimo-eth/daimo/assets/169280/a9f2072c-953c-47d5-b11d-f941449acccc">

Reply:
i think i see what you mean. you're talking about getting rid of `HistoryOpScreen` (below) and linking directly to `AccountScreen`

two thoughts:

1. HistoryOpScreen could def use a design pass
2. I'm down to bypass it --tap on HomeScreen history, go direct to account.

then if you want the receipt for a particular transaction, you click once on `HomeScreen` > click on that same transaction again on the "Transactions between you" list on the `AccountScreen` > see transaction details

one challenge to solve: if i scroll down on HomeScreen to a transaction that happened a while ago, then I have to repeat that process on AccountScreen to find that same tx again.

LMK if you have ideas.

Reply:
@dcposch Yes, exactly—I was implying removing the HistoryOpScreen (above) and linking directly to AccountScreen. 

1. I'm happy to create new designs here for the HistoryOpScreen!
2. Regarding the challenge to solve, do we know if the transaction history is that valuable of a touchpoint to users to have this be a significant issue? I can see the added friction there if users have to find the transaction again, but this may not be as large of an issue we believe it may be.